templates/igt/generic-uboot-tftp-ramdisk-igt-template.jinja2

```diff
@@ -0,0 +1,7 @@
+{% extends 'boot/generic-uboot-tftp-ramdisk-template.jinja2' %}
```

Comment:
We're not actually generating this test for any U-Boot devices, are we?
(same thing for v4l2 I believe)

If not then it would seem worth adding these test plans to another device that runs mainline U-Boot as a reference, for example the ODROID-XU3 if it makes sense to run these tests on it.